app/assets/javascripts/spin.js

```diff
@@ -0,0 +1,377 @@
+/**
```

Comment:
Let's add it as a dependency, https://github.com/dnagir/spinjs-rails gem is available